src/components/Expenses.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+
+import Input from './Input';
+
+import './dashboard.css';
+
+const Expenses = () => {
+
+    // to do when connecting the api
+    const [expensesDetails] = useState({
+        name: 'food',
+        amount: '6000'
+    });
+
+    const [expenseDetails, setExpenseDetails] = useState({
+        name: '',
+        amount: ''
+    });
+
+    const onExpenseChange = (newState) => {
+        setExpenseDetails({ ...expenseDetails, ...newState });
+    };
+
+    return (
+        <>
+            <h1>EXPENSE DASHBOARD</h1>
+            <section id="finance-table">
+                <h2>ADDING MY EXPENSES</h2>
+                <div>
```

Comment:
Since you have another header below does it not make sense for this to be another sub section of the above one?